packages/iTwinUI-react/src/core/ButtonGroup/ButtonGroup.tsx

```diff
@@ -110,7 +110,7 @@ export const ButtonGroup = React.forwardRef<HTMLDivElement, ButtonGroupProps>(
             )}
 
           {visibleCount < items.length
-            ? items.slice(0, visibleCount - 1)
+            ? items.slice(0, visibleCount < 1 ? 0 : visibleCount - 1)
```

Comment:
No need for a change but just to note that you can also just write `Math.max(0, visibielCount - 1)`

Reply:
I was looking for that!!

Reply:
I think this will be more clear

Reply:
Updated it, thanks!